### PR TITLE
Migration to 3.4 #6585

### PR DESCRIPTION
**Pull request by priyankarani**

### CODE REVIEW

File: tests/test_project.py [hunk 1]
```diff
@@ -14,15 +14,13 @@
...
-CONFIG.options['data_path'] = '.'
-CONFIG['smtp_from'] = 'test@openlabs.co.in'
 from minimock import Mock
-
 from trytond.tests.test_tryton import DB_NAME, USER, CONTEXT
 from trytond.transaction import Transaction
-
 from test_base import TestBase
+from trytond.config import config
+config.set('database', 'path', '.')
```
Comment (prakashpp) [inline tests/test_project.py]: this should be default to `/tmp`

File: tests/test_task.py [hunk 2]
```diff
@@ -4,21 +4,20 @@
...
-from trytond.config import CONFIG
-CONFIG['smtp_from'] = 'test@openlabs.co.in'
-CONFIG['data_path'] = '.'
 from minimock import Mock
-
 from trytond.tests.test_tryton import POOL, DB_NAME, USER, CONTEXT
 from trytond.transaction import Transaction
 from test_base import TestBase
+from trytond.config import config
+config.set('database', 'path', '.')
```
Comment (prakashpp) [inline tests/test_task.py]: `/tmp`

File: tests/test_views_depends.py [hunk 3]
```diff
@@ -4,20 +4,21 @@
...
     :license: BSD, see LICENSE for more details.
 """
 import sys
 import os
+import unittest
+
+import trytond.tests.test_tryton
+from trytond.tests.test_tryton import test_view, test_depends
+
 DIR = os.path.abspath(os.path.normpath(os.path.join(
```
Comment (prakashpp) [inline tests/test_views_depends.py]: this is not needed anymore. Can you please remove all such changes please

Comment (priyankarani) [inline tests/test_views_depends.py]: This change is needed for some flake8 issues to be fixed.

Comment (prakashpp) [inline tests/test_views_depends.py]: no, remove this. This piece of code is doing nothing.

Comment (prakashpp) [inline tests/test_views_depends.py]: @priyankarani can you please remove this?

**Pull request merged**